pkg/webrtc/webrtc.go

```diff
@@ -198,21 +198,22 @@ func (w *WebRTC) StartClient(isMobile bool, iceCB OnIceCallback) (string, error)
 	})
 
 	w.connection.OnTrack(func(remoteTrack *webrtc.Track, receiver *webrtc.RTPReceiver) {
-		rtpBuf := make([]byte, 1400)
-
-		log.Println("Received Voice from Client")
-		for {
-			if w.RoomID == "" {
-				// skip sending voice when game is not running
-				continue
-			}
-
-			i, err := remoteTrack.Read(rtpBuf)
```

Comment:
Even if I move RoomID check after, it still caused high CPU. Need to check if there is buffer mechanism in remoteTrack